Changelog.md

```diff
@@ -2,6 +2,9 @@
 
 ## Ignition Common 3.X.X
 
+1. Fix hierarchical submesh processing in the ColladaLoader.
```

Comment:
Can you rename the PR to be consistent with this? That way the squash merge will be consistent.

Reply:
Done.